src/react/utils/index.js

```diff
@@ -42,3 +42,10 @@ export const stripTrailingSlash = name => name.slice(-1) === '/' ? name.slice(0,
 export const addTrailingSlash = name => name ? name.slice(-1) !== '/' ? `${name}/`: name : '';
 
 export const maybePluralize = (count, noun, suffix = 's') => `${count} ${noun}${count > 1 ? suffix : ''}`;
+
+export function stripQuotes(s) {
+    if (s.charAt(0) === '"' && s.charAt(s.length-1) === '"') {
```

Comment:
not sure if its any faster but I prefer to use startsWith and endsWith like:
```suggestion
    if (s.startsWith('"') && s.endsWith('"')) {
```